packages/web-components/src/accordion-item/accordion-item.stories.ts

```diff
@@ -54,6 +58,7 @@ export default {
       description: 'Controls the size of the Accordion Item.',
       name: 'size',
       options: ['', ...Object.values(AccordionItemSize)],
+      mapping: { '': null, ...AccordionItemSize },
```

Comment:
Comment for anyone looking, maps the options for the select so that if the value is unset it returns null and removes the attribute completely from the code preview.

`<fluent-accordion-item size="medium"></fluent-accordion-item>`

Then unsetting `size`:

`<fluent-accordion-item></fluent-accordion-item>`

VS

`<fluent-accordion-item size=""></fluent-accordion-item>`